src/slack/slack.js

```diff
@@ -100,8 +100,32 @@ export class Slack {
 
     // console.log('Upgrade complete')
 
-    // TODO: inline handleMessage
-    this.boltApp.event('message', ({event}) => this.handleMessage(event))
+    this.boltApp.event('message', async ({event, say}) => {
+      logger.info('message event')
+
+      if (event.subtype) return
+
+      if (this.amIMentioned(event)) {
+        try {
```

Comment:
we now have try-catch both up here (which is in line with the action handling) and inside the `handleUserMessage`. I like the consistency, so maybe we can get rid of the inside try-catch?